sources/smallcaps.fea

```diff
@@ -0,0 +1,309 @@
+    sub \ograve by \ograve.sc ;
+    sub \oacute by \oacute.sc ;
+    sub \ocircumflex by \ocircumflex.sc ;
+    sub \otilde by \otilde.sc ;
+    sub \odieresis by \odieresis.sc ;
+    sub \oslash by \oslash.sc ;
+    sub \ugrave by \ugrave.sc ;
+    sub \uacute by \uacute.sc ;
+    sub \ucircumflex by \ucircumflex.sc ;
+    sub \udieresis by \udieresis.sc ;
+    sub \yacute by \yacute.sc ;
+    sub \thorn by \thorn.sc ;
+    sub \ydieresis by \ydieresis.sc ;
+    sub \uni0101 by \uni0101.sc ;
+    sub \uni0103 by \uni0103.sc ;
+    sub \uni0105 by \uni0105.sc ;
+    sub \uni0107 by \uni0107.sc ;
+    sub \uni0109 by \uni0109.sc ;
+    sub \uni010B by \uni010B.sc ;
+    sub \uni010D by \uni010D.sc ;
+    sub \uni010F by \uni010F.sc ;
+    sub \uni0111 by \uni0111.sc ;
+    sub \uni0113 by \uni0113.sc ;
+    sub \uni0115 by \uni0115.sc ;
+    sub \uni0117 by \uni0117.sc ;
+    sub \uni0119 by \uni0119.sc ;
+    sub \uni011B by \uni011B.sc ;
+    sub \uni011D by \uni011D.sc ;
+    sub \uni011F by \uni011F.sc ;
+    sub \uni0121 by \uni0121.sc ;
+    sub \uni0123 by \uni0123.sc ;
+    sub \uni0125 by \uni0125.sc ;
+    sub \uni0127 by \uni0127.sc ;
+    sub \uni0129 by \uni0129.sc ;
+    sub \uni012B by \uni012B.sc ;
+    sub \uni012D by \uni012D.sc ;
+    sub \uni012F by \uni012F.sc ;
+    sub \dotlessi by \dotlessi.sc ;
+    sub \uni0133 by \uni0133.sc ;
+    sub \uni0135 by \uni0135.sc ;
+    sub \uni0137 by \uni0137.sc ;
+    sub \uni013A by \uni013A.sc ;
+    sub \uni013C by \uni013C.sc ;
+    sub \uni013E by \uni013E.sc ;
+    sub \uni0140 by \uni0140.sc ;
+    sub \lslash by \lslash.sc ;
+    sub \uni0144 by \uni0144.sc ;
+    sub \uni0146 by \uni0146.sc ;
+    sub \uni0148 by \uni0148.sc ;
+    sub \uni0149 by \uni0149.sc ;
+    sub \uni014D by \uni014D.sc ;
+    sub \uni0151 by \uni0151.sc ;
+    sub \oe by \oe.sc ;
+    sub \uni0155 by \uni0155.sc ;
+    sub \uni0157 by \uni0157.sc ;
+    sub \uni0159 by \uni0159.sc ;
+    sub \uni015B by \uni015B.sc ;
+    sub \uni015D by \uni015D.sc ;
+    sub \uni015F by \uni015F.sc ;
+    sub \scaron by \scaron.sc ;
+    sub \uni0163 by \uni0163.sc ;
+    sub \uni0165 by \uni0165.sc ;
+    sub \uni0167 by \uni0167.sc ;
+    sub \uni0169 by \uni0169.sc ;
+    sub \uni016B by \uni016B.sc ;
+    sub \uni016D by \uni016D.sc ;
+    sub \uni016F by \uni016F.sc ;
+    sub \uni0171 by \uni0171.sc ;
+    sub \uni0173 by \uni0173.sc ;
+    sub \uni0175 by \uni0175.sc ;
+    sub \uni0177 by \uni0177.sc ;
+    sub \uni017A by \uni017A.sc ;
+    sub \uni017C by \uni017C.sc ;
+    sub \zcaron by \uni017E.sc ;
+    sub \uni01F0 by \uni01F0.sc ;
+    sub \uni01FB by \uni01FB.sc ;
+    sub \uni01FD by \uni01FD.sc ;
+    sub \uni01FF by \uni01FF.sc ;
+    sub \uni203C by \uni203C.sc ;
+    sub \uni2047 by \uni2047.sc ;
+    sub \uni214B by \uni214B.sc ;
+    sub \scommaaccent by \scommaaccent.sc ;
+    sub \uni021B by \uni021B.sc ;
+} smcpLowercasetoSmallCapitalslookup0;
+
+feature c2sc {
```

Comment:
Removes these feature blocks. The languages and scripts are declared globally to ensure consistency. Inconsistencies in these can mean some lookups are not applied for some languages.